### DISCUSSION TITLE
Added: LDM Notes from Apr 15, 2019

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2447] [General]
https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-04-15.md

1. CancellationToken in iterators
2. Implied nullable constraints in nullable disabled code
3. Inheriting constraints in nullable disabled code
4. Declarations with constraints declared in #nullable disabled code
5. Result type of `??=` expression
6. Use annotation context to compute the annotations?
7. Follow-up decisions for pattern-based Index/Range

**Comment by NetMage** [comment]
Is there any place to see the discussion behind the decision to require pattern-based Range to not accept extension Slice methods?
